extensions/netty/runtime/pom.xml

```diff
@@ -39,6 +39,17 @@
             <groupId>com.oracle.substratevm</groupId>
             <artifactId>svm</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-transport-native-epoll</artifactId>
+            <classifier>linux-x86_64</classifier>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-transport-native-kqueue</artifactId>
+            <classifier>osx-x86_64</classifier>
+        </dependency>
```

Comment:
Maybe leave these dependencies to be manually added in the project if the property is set? Not sure it makes sense to include the OSX and Linux dependencies in `quarkus-netty`

Reply:
It's perfectly safe to include these dependencies and they are ignored in Windows. Both Spring Boot and Wildfly have them baked in (the later in netty-all). We could do as yu suggest but then we would have to add some documentation.